### PR TITLE
Add Codeberg (codeberg.org)

### PR DESCRIPTION
**Pull request by seabass-labrax**
This commit adds [Codeberg](https://codeberg.org), a code hosting site intended for free software developers.

[SVGO](https://github.com/svg/svgo) was not able to minify the SVG logo effectively; it caused the left side of the logo to be cut off. However, I have used [Tidy](https://www.html-tidy.org/) to remove unnecessary whitespace, which did result in a significant reduction in file size as compared to the original.

### CODE REVIEW

**Comment by Carlgo11** [comment]
Try [svgomg](https://jakearchibald.github.io/svgomg/) and change the precision so that the image looks okay. Then enable all the *features* checkboxes and multipass.

File: img/c/codeberg.org.svg [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+<path transform="scale(0.26458333)"
+d="m 42519.285,-7078.7891 a 0.76086879,0.56791688 0 0 0 -0.738,0.6739 l 33.586,125.8886 a 87.182358,87.182358 0 0 0 39.381,-33.7636 l -71.565,-92.5196 a 0.76086879,0.56791688 0 0 0 -0.664,-0.2793 z"
+style="font-variation-settings:normal;opacity:0.5;vector-effect:none;fill:url(#linearGradient6918);fill-opacity:1;stroke:none;stroke-width:3.67846;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:2;stroke-dasharray:none;stroke-dashoffset:0;stroke-opacity:1;paint-order:stroke markers fill;stop-color:#000000;stop-opacity:1"
+id="path6733" />
+<path d="m 42517.32,-7119.4805 a 87.182358,87.182358 0 0 0 -86.855,87.1817 87.182358,87.182358 0 0 0 13.312,46.3047 l 72.688,-93.9727 a 1.3609979,1.0158567 0 0 1 2.359,0 l 72.692,93.9766 a 87.182358,87.182358 0 0 0 13.314,-46.3086 87.182358,87.182358 0 0 0 -87.182,-87.1817 87.182358,87.182358 0 0 0 -0.328,0 z"
+style="opacity:1;fill:#2185d0;fill-opacity:1;stroke:none;stroke-width:2.01453px;stroke-opacity:1;paint-order:stroke markers fill;stop-color:#000000"
+id="circle6810" transform="scale(0.26458333)" />
+</g>
+</g>
+</svg>
```
Comment (Carlgo11) [inline img/c/codeberg.org.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 4.21 4.21"><defs><linearGradient xlink:href="#a" id="b" x1="42519.29" x2="42575.34" y1="-7078.79" y2="-6966.93" gradientTransform="translate(-1025.54 172.08) scale(.02417)" gradientUnits="userSpaceOnUse"/><linearGradient id="a"><stop offset="0" stop-color="#2185d0" stop-opacity="0"/><stop offset=".5" stop-color="#2185d0" stop-opacity=".49"/><stop offset="1" stop-color="#2185d0" stop-opacity=".63"/></linearGradient></defs><path fill="url(#b)" d="M2.15.98a.02.01 0 0 0-.02.02l.81 3.04a2.1 2.1 0 0 0 .95-.81L2.16.99a.02.01 0 0 0-.01 0z" opacity=".5"/><path fill="#2185d0" d="M2.1 0A2.1 2.1 0 0 0 0 2.1a2.1 2.1 0 0 0 .32 1.13L2.08.96a.03.02 0 0 1 .06 0l1.75 2.27a2.1 2.1 0 0 0 .32-1.12A2.1 2.1 0 0 0 2.11 0a2.1 2.1 0 0 0-.01 0z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by seabass-labrax** [comment]
Say, thank you for the suggested changes @Carlgo11! I did try to use svgomg, but unfortunately it wouldn't run at all in my web browser.

**Review by Carlgo11 — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]

**Comment by seabass-labrax** [comment]
Thanks for merging! :)

**Pull request merged**